### PR TITLE
feat: Add a button to create a note from Drive

### PR DESCRIPTION
**Pull request by Crash--**
Add a button in the MoreMenuAction on Drive to create a Note. 

- [x] Need an upgrade of Cozy-Client (https://github.com/cozy/cozy-client/pull/647/files) 
- [x] Upgrade FR transifex 
- [x] Waiting for the SVG

### CODE REVIEW

File: src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  )
+
+  return {
+    fetchStatus,
+    isInstalled,
+    url
+  }
+}
+
+export default useAppLinkWithStoreFallback
```
Comment (y-lohse) [inline src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx]: Reminds me of something 🤓

File: package.json [hunk 2]
```diff
@@ -112,6 +112,7 @@
     "cozy-scanner": "0.2.0",
     "cozy-scripts": "1.13.2",
     "cozy-sharing": "^1.8.0",
+    "cozy-stack-client": "11.2.0",
```
Comment (y-lohse) [inline package.json]: This is not a dev dep, right? We didn't add it only for tests?

The main problem I see with this is that we now need to upgrade both packages manually :(

File: src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx (hunk 1 above)
Comment (Crash--) [reply src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx]: Let's me add it in Cozy-Client ;).

File: package.json (hunk 2 above)
Comment (Crash--) [reply package.json]: Nope. This is not for the test :/

File: src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx (hunk 1 above)
Comment (Crash--) [reply src/drive/web/modules/drive/Toolbar/components/useAppLinkWithStoreFallBack.jsx]: https://github.com/cozy/cozy-client/pull/651

**Pull request merged**